docs/topics/auth/default.txt

```diff
@@ -961,6 +961,13 @@ implementation details see :ref:`using-the-views`.
     * ``extra_context``: A dictionary of context data that will be added to the
       default context data passed to the template.
 
+    * ``redirect_authenticated_user``: Boolean. If True, authenticated users
+      accessing the login page will be redirected to
+      :setting:`settings.LOGIN_REDIRECT_URL <LOGIN_REDIRECT_URL>` or the ``next``
```

Comment:
On further review I think "``next`` argument" is wrong too; there is no such argument to the view, it's really a querystring var. Could update to say that, but I'm wondering if it might be best to instead just say "will be redirected as if they had just successfully logged in."